src/Col.tsx

```diff
@@ -36,9 +36,9 @@ export interface ColProps
   lg?: ColSpec;
   xl?: ColSpec;
   xxl?: ColSpec;
+  [key: string]: any;
```

Comment:
This is rough -- `[key: string]: any` allows pretty much any prop.  But I don't think there's any good way to avoid it.  These are top-level properties so `[key: string]: ColSpec` would produce an incompatible index signature error.